### PR TITLE
To-do list project

### PR DESCRIPTION
**Pull request by igorkol91**
In this project, we created an app that can create projects and add separate to-do's on each of them.
- Added creation of projects that contains their name only
- Added an option for removing project's
- When clicked on a project we get a list of todo's of that specific project
- Also new to-do form will appear on the right side of the screen for creating a new one
- Added option for selecting todo's and editing them
- On the to-do, we can mark them as completed change their priority, or delete them

### CODE REVIEW

**Review by BrianSammit — CHANGES_REQUESTED** [review]
CHANGES REQUIRED 🔴 🛑 
-------------
Hello @igorkol91 @barackm, great job you did on this project 💪 
TO HIGHLIGHT: 

- Your project is professional 
- Your repo is professional 
- Your project is separated into modules
- Well structured files 
- I like the design of the project
- Your project is working properly

However, there are minor changes to make to complete this project 🛠️ see them below 👀 👇 
-------------

- [ ] Please use arrow functions instead of the `function` keyword. In `dom.js` `editTodo.js` `todo.js` you have functions keyword instead of arrow function. Please change them. 
- [ ] When I run `npm start` I got the directories files instead of the project. Please fix this. 

Capture

---------------
Make these requirements and send it again for another review 💯 👍 
HAPPY CODING!!! 

tenor-1

File: src/dom.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    }
+  };
+
+  function updateDefaultTodo(id) {
+    const allTodos = JSON.parse(localStorage.getItem('todoList')) || [];
+    const todo = allTodos.find((t) => t.id.toString() === id.toString());
+    defaultTodo = todo;
+    editTodo(defaultTodo, defaultProject);
+    updateDefaultProject(defaultProject.id);
+  }
```
Comment (SajjadAhmad14) [inline src/dom.js]: Please use the arrow function here.

File: src/helper.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { v4 as uuidv4 } from 'uuid';
+import render from './dom';
+
+function NewProject(name) {
+  this.id = uuidv4();
+  this.name = name;
+}
```
Comment (SajjadAhmad14) [inline src/helper.js:7]: Please use the arrow function here.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, you have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
- [ ] Git flow is used properly. 
- [ ] Project has a fascinating UI.
## To Improve 🙏
- [ ] Please remove function keywords from all the modules and use ES6 arrow functions instead.
### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

File: src/helper.js (hunk 2 above)
Comment (barackm) [reply src/helper.js:7]: We added the function keyword because constructor functions can not be arrow functions.

**Comment by barackm** [comment]
We added the function keyword because constructor functions can not be arrow functions.

**Review by cyonii — APPROVED** [review]
### STATUS: APPROVED ✅

Hello team, your project has now met the minimum requirement, and is hereby approved.

You have a really cool project.
Taking the time give your app a neat design is so impressive. 👏🔥😎

You can merge this PR and move on.
Happy coding.

> #### <br>Feel free to leave a comment or question if something is not clear.<br> Slack: @[CY Kalu](https://microverse-students.slack.com/team/U01494U96FR)<br><br>

**Review by cyonii — APPROVED** [review]

**Pull request merged**